html/package.json

```diff
@@ -34,20 +34,24 @@
   "typings": "lib/cjs/index.d.ts",
   "types": "lib/cjs/index.d.ts",
   "dependencies": {
+    "@cycle/dom": ">=20.4.0",
     "@cycle/run": "*",
-    "@cycle/dom": ">=19.3.0",
-    "snabbdom": "0.7.0",
-    "snabbdom-to-html": "3.3.0",
+    "snabbdom": "0.7.1",
+    "snabbdom-to-html": "5.1.1",
```

Comment:
What were all the breaking changes in snabbdom-to-html from 3.3.0 to 5.1.1? (I'm reviewing this but I don't know which commit exactly made this change) I'd assume this kind of change needs a `fix()` commit with clear breaking change notes.

Reply:
Update to snabbdom 0.7.0 and correct escaping of html inside `text`.

Reply:
Done